src/toil/common.py

```diff
@@ -876,6 +878,9 @@ def _runMainLoop(self, rootJob):
         :param toil.job.Job rootJob: The root job for the workflow.
         :rtype: Any
         """
+        logger.info("Running Toil version %s.", version)
+        logger.info("Configuration:\n %s", pformat(self.config.__dict__))
```

Comment:
Can you pull this out into a method called `logProcessContext()` in `src/toil/__init__.py`? 

That would allow us to add more logging in the future without having to modify two places.